deploy/modules/index.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, *args, **kwargs):
 
 	@exposed
 	def index(self, *args, **kwargs):
+		"""
+		The first two lines of code here ensure that requesting a non-existent module or template will throw a 404
+		instead of referring to index. Remove them if you wish to alter this behaviour.
+		"""
+		if len(args) >= 1:
```

Comment:
I think we can check the kwargs as well.
```suggestion
		if args or kwargs:
```